bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html

```diff
@@ -29,33 +32,46 @@ <h1 class="title-shadow-box">Mozilla CA Certificate Policy</h1>
     to verify certificates for SSL servers and S/MIME email users without having
     to ask users for further permission or information.</p>
   <p>This is the official Mozilla policy for CA certificates that are distributed
-    with Mozilla software products. This policy consists of the following three
+    with Mozilla software products. This policy consists of the following
     sections:</p>
   <ul>
     <li>
-      <p><a href="#inclusion">Applying for Inclusion of Root Certificates in Mozilla Products</a>
-      </p>
+      <p><a href="audit-requirements">Audit Requirements</a></p>
+      <p>Before being included and periodically thereafter, CAs are required to
+        obtain certain audits. This section describes the requirements for those
+        audits.</p>
+    </li>
+    <li>
+      <p><a href="#applying-for-inclusion-of-root-certificates-in-mozilla-products">Applying for
+        Inclusion of Root Certificates in Mozilla Products</a></p>
       <p>This section describes the obligations of Certification Authorities
         applying for inclusion of their root certificates in Mozilla Products.
         This includes considerations that are taken into account such as the CA’s
         publicly available documentation about their policies, and audits of the
         CA’s operations in support of the documented policies.</p>
     </li>
     <li>
-      <p><a href="#maintenance">Maintaining Confidence in Included Root Certificates</a></p>
+      <p><a href="#maintaining-confidence-in-included-root-certificates">Maintaining Confidence in
```

Comment:
These anchor links are pretty long, suggest something shorter like it used to be i.e. `#maintenance`?